framework/src/meshgenerators/ModifyNodeGenerator.C

```diff
@@ -0,0 +1,54 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ModifyNodeGenerator.h"
+
+registerMooseObject("MooseApp", ModifyNodeGenerator);
+
+InputParameters
+ModifyNodeGenerator::validParams()
+{
+  InputParameters params = MeshGenerator::validParams();
+
+  params.addRequiredParam<MeshGeneratorName>("input", "Input mesh to modify");
+
+  params.addRequiredParam<std::vector<int>>("node_id", "Id of modified node");
```

Comment:
Suggest changing `int` to `dof_id_type` to be consistent with its use in `Mesh::query_node_ptr`.